configs/components/oifs/change_icm_date.sh

```diff
@@ -63,17 +31,28 @@ cdo -V
 echo " "
 echo " Input dir: $indir "
 echo " Output dir: $outdir "
-echo " Exp ID: $expid "
+echo " InExp ID: $inexpid "
+echo " OutExp ID: $outexpid "
 echo " Start date: $startdate "
 echo " End date: $enddate "
-
+echo " Perturb: $perturb "
+# Miguel: Style is not supported anymore in this preprocessing scripts, I don't
+# understand why it's being removed. @JAN, @JOAKIM?
```

Comment:
I think this is because Joakim took back his own long run changes in the source code and decided to use the esm_tools based ones now. Maybe @joakimkjellsson or @seb-wahl can comment?

Reply:
Correct! "style" was used becuase Jan and I had different ways of coping with the long runs. 
But now that we are both on the same page (i.e. we are both "jesus") I think we can remove the "style" variable. 
Cheers
Joakim